docs/source/create_channel/create_channel.md

```diff
@@ -0,0 +1,383 @@
+# Creating a new channel
+
+You can use this tutorial to learn how to create new channels using the [configtxgen](../commands/configtxgen.html) CLI tool and then use the [peer channel](../commands/peerchannel.html) commands to join a channel with your peers. We will create the new channel using the Fabric test network. As a result, developers can use this tutorial to create custom channels for development and testing. However, the steps in this tutorial can also be used by network operators who are creating channels on a production network.
+
+In the process of creating the channel, this tutorial will take you through the following steps and concepts:
+
+- [Setting up the configtxgen tool](#setting-up-the-configtxgen-tool)
+- [Using the configtx.yaml file](#the-configtx-yaml-file)
+- [The orderer system channel](#the-orderer-system-channel)
+- [Creating an application channel](#creating-an-application-channel)
+- [Joining peers to the channel](#join-peers-to-the-channel)
+- [Setting anchor peers](#set-anchor-peers)
+
+## Setting up the configtxgen tool
+
+Channels are created by creating a channel genesis block that contains the initial configuration of the new channel. While it is possible to build these blocks manually, it is easier to use the [configtxgen](../commands/configtxgen.html) tool to write genesis blocks and channel creation artifacts. The tool works by reading a `configtx.yaml` file that defines the configuration of your channel. Before we discuss the `configtx.yaml` file in the next section, we can get started by downloading and setting up the `configtxgen` tool.
```

Comment:
..created by creating.... 
In order to create a channel you need to provide a channel genesis block that contains the initial configuration of the new channel.

Reply:
Channels are created by building a .... 

You never need to provide an application channel genesis block, since that is created by the ordering service. One reason for slightly elliptical phrases from time to time.